staging/src/k8s.io/client-go/discovery/cached/disk/cached_discovery_test.go

```diff
@@ -210,14 +215,400 @@ func TestOpenAPIDiskCache(t *testing.T) {
 				// Ensure schema call is still served from disk
 				_, err = newPaths[k].Schema(contentType)
 				assert.NoError(t, err)
-				assert.Equal(t, i, fakeServer.RequestCounters["/openapi/v3"])
 				assert.Equal(t, 1, fakeServer.RequestCounters[path])
 			}
 		})
 	}
 
 }
 
+// Tests function "ServerGroups" when the "unaggregated" discovery is returned.
+func TestMemCacheGroupsAndMaybeResources(t *testing.T) {
```

Comment:
These two lines don't match, and the test actually calls `ServerGroups`, so the comment is right but the function name isn't.